eng/versioning/external_dependencies.txt

```diff
@@ -32,18 +32,18 @@ commons-codec:commons-codec;1.13
 io.micrometer:micrometer-core;1.2.0
 io.micrometer:micrometer-registry-azure-monitor;1.2.0
 io.micrometer:micrometer-registry-graphite;1.2.0
-io.netty:netty-buffer;4.1.49.Final
-io.netty:netty-codec-http;4.1.49.Final
-io.netty:netty-codec-http2;4.1.49.Final
-io.netty:netty-handler;4.1.49.Final
-io.netty:netty-handler-proxy;4.1.49.Final
+io.netty:netty-buffer;4.1.51.Final
+io.netty:netty-codec-http;4.1.51.Final
+io.netty:netty-codec-http2;4.1.51.Final
+io.netty:netty-handler;4.1.51.Final
+io.netty:netty-handler-proxy;4.1.51.Final
 io.netty:netty-tcnative;2.0.29.Final
 io.netty:netty-tcnative-boringssl-static;2.0.29.Final
```

Comment:
Looks like the newer versions of `netty-handler` use version `2.0.31` of these artifacts.

Reply:
good catch @alzimmermsft - I will update those as well.